src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyleRecordsAndCompactCtors.java

```diff
@@ -69,38 +69,38 @@ public record MySecondRecord() {
          * Public check should fail</code> // violation and line below, too
          * should fail <
          */
-        public void method8() {
+        public void method8() { // violation
         }
     }
 
     /**
      *
      */
-    public record MyThirdRecord(String myString) {
+    public record MyThirdRecord(String myString) { // ok
     }
 
-    public record MyFourthRecord(String myString) {
+    public record MyFourthRecord(String myString) { // ok
         /**
          * This Javadoc contains unclosed tag.
          * <code>unclosed 'code' tag<code> // violation
          */
-        private static void unclosedTag() {
+        private static void unclosedTag() { // violation
             System.out.println("stuff");
         }
 
-        public MyFourthRecord {
+        public MyFourthRecord { // ok
             /**
-             * No period at the end of this sentence // violation
+             * No period at the end of this sentence // ok
              */
-            String myOtherString = "mystring";
+            String myOtherString = "mystring"; // ok
         }
     }
 
-    public record MyFifthRecord() {
+    public record MyFifthRecord() { // ok
         /**
          * No period here on compact ctor // violation
          */
-        public MyFifthRecord {
+        public MyFifthRecord { // violation
```

Comment:
check `// violation` comments and expected violation messages they mismatch

Reply:
Like here only period is missing. If I don't write `//violation` here and When I running the verify lifecycle. I am getting error because it expects `// violation` and `// ok`.

Reply:
done